src/index.js

```diff
@@ -5,10 +5,10 @@ function component() {
     const element = document.createElement('div');
 
     // Lodash, now imported by this script
-    element.innerHTML = 'Hello world!';
-    element.classList.add('hello');
+    // element.innerHTML = 'Goodbie world!';
+    // element.classList.add('hello');
 
-    return element;
+    // return element;
 }
 
 document.body.appendChild(component());
```

Comment:
@ValGorovyi Remove this code completely. It appends undefined to the body. It's wrong even if it does nothing